picard/formats/asf.py

```diff
@@ -186,8 +186,11 @@ def _save(self, filename, metadata):
         log.debug("Saving file %r", filename)
         file = ASF(encode_filename(filename))
 
-        if config.setting['clear_existing_tags']:
-            file.tags.clear()
+        if config.setting["clear_existing_tags"]:
```

Comment:
From what I can see in this PR the calls in the various formats are always the same and that could go into the `file.remove_tags()` method:

```python
if config.setting["clear_existing_tags"]:
    if config.setting["remove_extra_padding"]:
        tags.delete()
    else:
        tags.clear()
```